deploy/examples/create-external-cluster-resources.py

```diff
@@ -375,6 +385,8 @@ def _convert_hostname_to_ip(self, host_name):
         return ip
 
     def get_active_and_standby_mgrs(self):
+        if self._arg_parser.dry_run:
+            return "", self.dry_run("ceph status")
```

Comment:
```suggestion
            return "", self.dry_run("ceph status")
```
from this `""""` we are getting `None`

Reply:
We don't want that None to be displayed.